deploy/charts/cert-manager/templates/networkpolicy-egress.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.global.networkPolicyEnabled }}
+
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ template "webhook.fullname" . }}-allow-egress
+  namespace: {{ include "cert-manager.namespace" . }}
+spec:
+  egress:
+  - ports:
+    - port: 80
+      protocol: TCP
+    - port: 443
+      protocol: TCP
+    - port: 53
+      protocol: TCP
+    - port: 53
+      protocol: UDP
```

Comment:
I agree these are sane defaults. Would someone ever want to change this? Perhaps they would want to block 80 TCP and 53 UDP?

Reply:
Maybe. There a lot of interesting people out there :D externalised.